.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: php
+php:
+  - '8.0'
```

Comment:
Let's develop for all [supported versions](https://www.php.net/supported-versions.php)

```suggestion
  - '7.3'
  - '7.4'
  - '8.0'
```